src/sources/apache_metrics/parser.rs

```diff
@@ -0,0 +1,852 @@
+                    value: MetricValue::Gauge { value: 1.0 },
+                },
+                Metric {
+                    name: "apache_scoreboard".into(),
+                    timestamp: Some(now),
+                    tags: Some(map! {"state" => "open"}),
+                    kind: MetricKind::Absolute,
+                    value: MetricValue::Gauge { value: 325.0 },
+                },
+                Metric {
+                    name: "apache_scoreboard".into(),
+                    timestamp: Some(now),
+                    tags: Some(map! {"state" => "reading"}),
+                    kind: MetricKind::Absolute,
+                    value: MetricValue::Gauge { value: 1.0 },
+                },
+                Metric {
+                    name: "apache_scoreboard".into(),
+                    timestamp: Some(now),
+                    tags: Some(map! {"state" => "sending"}),
+                    kind: MetricKind::Absolute,
+                    value: MetricValue::Gauge { value: 1.0 },
+                },
+                Metric {
+                    name: "apache_scoreboard".into(),
+                    timestamp: Some(now),
+                    tags: Some(map! {"state" => "starting"}),
+                    kind: MetricKind::Absolute,
+                    value: MetricValue::Gauge { value: 1.0 },
+                },
+                Metric {
+                    name: "apache_scoreboard".into(),
+                    timestamp: Some(now),
+                    tags: Some(map! {"state" => "waiting"}),
+                    kind: MetricKind::Absolute,
+                    value: MetricValue::Gauge { value: 64.0 },
+                },
+                Metric {
+                    name: "apache_up".into(),
+                    timestamp: Some(now),
+                    tags: None,
+                    kind: MetricKind::Absolute,
+                    value: MetricValue::Gauge { value: 1.0 },
+                },
+                Metric {
+                    name: "apache_uptime_seconds_total".into(),
+                    timestamp: Some(now),
+                    tags: None,
+                    kind: MetricKind::Absolute,
+                    value: MetricValue::Counter { value: 12.0 },
+                },
+                Metric {
+                    name: "apache_workers".into(),
+                    timestamp: Some(now),
+                    tags: Some(map! {"state" => "busy"}),
+                    kind: MetricKind::Absolute,
+                    value: MetricValue::Gauge { value: 1.0 },
+                },
+                Metric {
+                    name: "apache_workers".into(),
+                    timestamp: Some(now),
+                    tags: Some(map! {"state" => "idle"}),
+                    kind: MetricKind::Absolute,
+                    value: MetricValue::Gauge { value: 74.0 },
+                },
+            ]
+        );
+        assert_eq!(errors.len(), 0);
+    }
+
+    #[test]
+    fn test_extended() {
```

Comment:
I'll update with code comments, but the difference between this and the above test is that the output is different depending on whether `ExtendedStatus` is enabled or not.

https://httpd.apache.org/docs/2.4/mod/core.html#extendedstatus